ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/VulnReportsPage.tsx

```diff
@@ -419,10 +430,17 @@ function VulnReportsPage() {
                                                         </Link>
                                                     </Td>
                                                     <Td>
-                                                        {
-                                                            report.resourceScope.collectionScope
-                                                                .collectionName
-                                                        }
+                                                        {isCollectionsRouteEnabled ? (
+                                                            <Link
+                                                                to={generatePath(collectionsPath, {
+                                                                    collectionId,
+                                                                })}
```

Comment:
@zhenpesky When a user clicks on the collection name in the reports table, should we send them directly to the page or open up a new tab and send them to the page?

Reply:
@sachaudh we should be able to utilize the same behavior to show the modal from clicking the "View" button < "Collection name" < "configure report scope" in step 1 of the wizard.

Reply:
I pushed this change and updated the PR description.